kernel-laws/shared/src/test/scala/cats/kernel/laws/LawTests.scala

```diff
@@ -132,7 +132,7 @@ class Tests extends AnyFunSuiteLike with Discipline {
   checkAll("Eq[List[HasEq[Int]]]", EqTests[List[HasEq[Int]]].eqv)
```

Comment:
I couldn't find a version specific version of this test (to test the `LazyList` instances)?

Reply:
Hmm, I know I grepped `tests` but I may have missed these. If so will add them now.

Reply:
Thanks, just pushed a commit adding these.